spec/models/piece_spec.rb

```diff
@@ -1,5 +1,67 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'obstructed method' do
+    it 'should return nil if invalid move' do
+      game = FactoryGirl.create(:game)
+      piece = FactoryGirl.create(:piece, x_position: 4, y_position: 4, type: 'Rook', game: game)
+      expect(piece.obstructed?(1, 3)).to eq nil
+    end
+
+    it 'should return false if not obstructed on right' do
+      game = FactoryGirl.create(:game)
```

Comment:
Since you're using game in every test, we can DRY this up a bit by doing the following.

At the top of the file

`let(:game) { FactoryGirl.create(:game) }`

Then `game` will be available in every test.

more info: http://betterspecs.org/#let